src/main/scala/Core/config.scala

```diff
@@ -193,7 +199,32 @@ case class FlexpretConfiguration(
 #define GPO_SIZES       {${gpoPortSizes.mkString(",")}}
 
 #endif
-    """
+
+"""
+  }
+
+  def generateMakeConfig() : String = {
+    return s"""
+#
+# This flexpret core config file is auto-generated, and based on the
+# configuration used to build the flexpret emulator.
+#
+# Do not edit.
+#
+#
+
+HAVE_THREADS := ${threads}
+HAVE_FLEXPRET := ${flex}
+HAVE_ISPM_KBYTES := ${imemConfig.sizeKB}
+HAVE_DSPM_KBYTES := ${dMemKB}
+HAVE_STACKSIZE := ${stackSize}
+HAVE_MUL := ${mul}
+
+# FIXME: Both set statically for now.
```

Comment:
This FIXME should maybe not be printed into the generated Makefile?

Reply:
Yes, I will fix that!